modules/machineset-yaml-rhv.adoc

```diff
@@ -44,25 +44,27 @@ spec:
           cluster_id: <ovirt_cluster_id> <6>
           template_name: <ovirt_template_name> <7>
           instance_type_id: <instance_type_id> <8>
-          cpu: <9>
-            sockets: <number_of_sockets> <10>
-            cores: <number_of_cores> <11>
-            threads: <number_of_threads> <12>
-          memory_mb: <memory_size> <13>
-          guaranteed_memory_mb:  <memory_size> <14>
-          os_disk: <15>
-            size_gb: <disk_size> <16>
-            storage_domain_id: <storage_domain_UUID> <17>
-          network_interfaces: <18>
-            vnic_profile_id:  <vnic_profile_id> <19>
+          sparse: <boolean value> <9>
+          format: <raw or cow> <10>
```

Comment:
Please add underscores to this variable:

>            format: <raw_or_cow> <10>

My apologies if I missed this in my initial review.